wagtail/contrib/forms/views.py

```diff
@@ -158,7 +158,7 @@ class SubmissionsListView(SpreadsheetExportMixin, BaseListingView):
     template_name = "wagtailforms/submissions_index.html"
     context_object_name = "submissions"
     form_page = None
-    ordering = ("-submit_time",)
+    default_ordering = ("-submit_time",)
```

Comment:
In order to fully reuse `BaseListingView.get_queryset`, we need to conform with the change in 4d435c5.